website/data/docs-navigation.js

```diff
@@ -90,6 +90,7 @@ export default [
       'manage-users-groups',
       'manage-roles',
       'manage-sessions',
+			'manage-ssh',
```

Comment:
I don't like the title "manage ssh" -- it's not managing SSH at all, it's making it easy to do a very specific ssh client workflow.

Reply:
Agree. I was trying my hardest to find a way to make it work with the existing `manage-` titles but alas, this one is too different. How about `workflow-ssh`?